### PR TITLE
Component coverage

### PR DESCRIPTION
**Pull request by miriam30**
# Summary 

Add coverage component

## 🖇 Related Links 

https://trello.com/c/aZqcLIZu/5-coverage-cobertura

## 🔀 Changes 
Add new color variables
Add component Coverage (map and text)

## 🛠 Testing 

Pull Branch
Go to coverage section
Review that map works and component is responsive

Screen Shot 2020-06-14 at 16 44 28
Screen Shot 2020-06-14 at 16 44 42
Screen Shot 2020-06-14 at 16 45 51

### CODE REVIEW

File: client/dist/sass/mains.scss [hunk 1]
```diff
@@ -12,3 +12,4 @@
 @import "sections/toolcards";
 @import "sections/toolsSlider";
 @import "sections/footer";
+@import "sections/coverage";
```
Comment (ElizaldeJE) [inline client/dist/sass/mains.scss:15]: Can we order this on how it loads?

File: client/dist/sass/sections/coverage.scss [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      &-section {
+        display: flex;
+      }
+
+      .dot {
+        width: 1.8rem;
+        height: 1.8rem;
+        display: flex;
+        border-radius: 50%;
+        background-color: #ffcb03;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:22]: Please use color variable.

File: client/dist/sass/sections/coverage.scss [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        .dot-hover {
+          width: fit-content;
+          min-width: 10rem;
+          height: fit-content;
+          padding: 0.3rem 1rem;
+          border-radius: 10.5px;
+          background-color: #7ac943;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:29]: Please use color variable.

File: client/dist/sass/sections/coverage.scss [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+          &::after {
+            content: "";
+            position: absolute;
+            top: 100%;
+            left: 25%;
+            margin-left: -5px;
+            border-width: 1rem;
+            border-style: solid;
+            border-color: #7ac943 transparent transparent transparent;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:47]: Please use the color variable.

File: client/dist/sass/sections/coverage.scss [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+          min-width: 10rem;
+          height: fit-content;
+          padding: 0.3rem 1rem;
+          border-radius: 10.5px;
+          background-color: #7ac943;
+          position: relative;
+          bottom: 4.5rem;
+          right: 2rem;
+          display: none;
+          font-size: 17px;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:34]: Do we have a font size for this?

File: client/dist/sass/sections/coverage.scss [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  &_description {
+    max-width: 45%
+  }
+
+  &-text {
+    font-size: $font-size-lw;
+    font-weight: normal;
+    line-height: 1.3;
+    text-align: left;
+    color: #4d4d4d;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:87]: Please use the color variable.

File: client/dist/sass/sections/coverage.scss [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+  }
+
+  &-title {
+    font-size: 51.3px;
+    font-weight: 900;
+    line-height: 0.98;
+    text-align: left;
+    color: #666666;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:95]: Please use the color variable.

File: client/dist/sass/sections/coverage.scss [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+  &-text {
+    font-size: $font-size-lw;
+    font-weight: normal;
+    line-height: 1.3;
+    text-align: left;
+    color: #4d4d4d;
+  }
+
+  &-title {
+    font-size: 51.3px;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:91]: Do we have a font variable for this?

File: client/dist/sass/sections/coverage.scss [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+@media(max-width: 650px) {
+  .rl_coverage {
+    display: block;
+    margin: 5rem 2.5rem 0;
+    height: auto;
+
+    &-text {
+      font-size: 1.4rem;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:142]: Do we have a font variable for this?

File: client/dist/sass/sections/coverage.scss [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+    }
+
+    &-title {
+      font-size: 2.8rem;
```
Comment (ElizaldeJE) [inline client/dist/sass/sections/coverage.scss:146]: Do we have a font variable for this?

File: client/src/LangData.js [hunk 11]
```diff
@@ -321,6 +326,11 @@ const projectBody = {
         backgroundColor: '#c0e6af'
       }
     ],
+    coverageObject: {
+      title: 'Cobertura',
+      mapImg: '../assets/images/coverage_map',
+      description: 'Nuestra gestión incluye una amplia diversidad de mandatos en la región latinoamericana, habiendo colaborado con organismos multilatelares, fondeadores internacionales, organismos de representación y supervisión, instituciones de la banca de desarrollo, así como sociedades cooperativas y financieras en varios países de la región; contribuyendo así con un modelo más eficiente y sustentable en beneficio de su población vulnerable.'
```
Comment (ElizaldeJE) [inline client/src/LangData.js:332]: Right here there's a typo it should be "multilaterales", not "multilatelares".

File: client/src/components/Coverage.jsx [hunk 12]
```diff
@@ -0,0 +1,41 @@
+import React,{ Component } from 'react'
+// import Fade from 'react-reveal/Fade'
```
Comment (ElizaldeJE) [inline client/src/components/Coverage.jsx:2]: do we need these comments here?

File: client/src/components/Coverage.jsx [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+// import Roll from 'react-reveal/Roll'
+import PropTypes from 'prop-types'
+
+export default class Slider extends Component {
```
Comment (ElizaldeJE) [inline client/src/components/Coverage.jsx:6]: Is this a proper component name? It's a bit vague and misleading.

File: client/src/components/Coverage.jsx [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+          <span className="dot dot-3"><span className="dot-hover">DOT 3</span></span>
+        </div>
+        <span className="dot"><span className="dot-hover">MEXICO</span></span>
+        <span className="dot"><span className="dot-hover">MEXICO</span></span>
+        <span className="dot"><span className="dot-hover">NICARAGUA</span></span>
+        <span className="dot"><span className="dot-hover">MEXICO</span></span>
+        <span className="dot"><span className="dot-hover">MEXICO</span></span>
+
+      </div>
+
```
Comment (ElizaldeJE) [inline client/src/components/Coverage.jsx:32]: Can we fix the indentation in this area please.

File: client/src/components/Coverage.jsx [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+      >
+      <div className="rl_coverage-map-dots">
+        <span className="dot"><span className="dot-hover">DOT 1</span></span>
+        <div className="rl_coverage-map-dots-section">
+          <span className="dot dot-2"><span className="dot-hover">DOT 2</span></span>
+          <span className="dot dot-3"><span className="dot-hover">DOT 3</span></span>
+        </div>
+        <span className="dot"><span className="dot-hover">MEXICO</span></span>
+        <span className="dot"><span className="dot-hover">MEXICO</span></span>
+        <span className="dot"><span className="dot-hover">NICARAGUA</span></span>
```
Comment (ElizaldeJE) [inline client/src/components/Coverage.jsx:27]: When you hover over this dot the country name overflows the container.

File: client/dist/sass/mains.scss (hunk 1 above)
Comment (miriam30) [reply client/dist/sass/mains.scss:15]: It has a label... The label is "Work in progress", your comments are irrelevant right now. I'll remove the label once it's ready to be reviewed

**Pull request merged**